### PR TITLE
Press4 464 | Adding Promotion option under marketing tab and products

### PR DESCRIPTION
**Pull request by manikantakailasa**
Add contextual placement areas for eCom addon 

1. Add promotions under WooCommerce Marketing tab
2. Add "Create a Promotion" under Product Data > General
3. Add "Create a Promotion" under Linked Products
4. Add "Promotions" as a list item under product data

story :https://jira.newfold.com/browse/PRESS4-464

### CODE REVIEW

File: includes/Promotions.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+var node = document.getElementById("custom_tab_data");
+node.innerHTML = `<div style="width: 100%;height:350px">
+    <p style=" text-align: center;padding-top:40px;line-height: 20px; font-size: 16px; font-weight: 600;">Boost Your Online Store Sales</p>
+    <p style=" text-align: center;line-height: 20px; font-size: 13px;">Maximize your sales by creating effective </br> promotions and campaigns like:</p>
+    <div style="text-align: center;">
+      <div style="display: flex;flex:1; align-items: center; justify-content:  space-around ;margin:0 auto; width: 50%;padding:20px 0">
+        <div style=" text-align: center;">
```
Comment (aratidgr8) [inline includes/Promotions.js:7]: @manikantakailasa can we move svg files under `src/icons` directory & import here?

Comment (manikantakailasa) [reply includes/Promotions.js:7]: with src path it is not able to find the files getting 404 error and import wont work in this file

**Review by aratidgr8 — CHANGES_REQUESTED** [review]

File: includes/Promotions.js (hunk 1 above)
Comment (manikantakailasa) [reply includes/Promotions.js:7]: path is correct not able to load it correctly

File: includes/ECommerce.php [hunk 2]
```diff
@@ -17,405 +17,479 @@
...
+  }
+
+	
+	/**
+	 * Add a Promotion button under Add New product tab
+   */
+	function custom_product_general_options() {
+		global $post;
+		$redirect_url =admin_url( 'admin.php?page='.$this->container->plugin()->id.'#/store/sales_discounts');
+		echo '<div class="options_group">';
```
[the remark this answers is not in the document]
Comment (manikantakailasa) [reply includes/ECommerce.php:450]: this is with the wordpress. if i remove this , content is appearing in all tabs.

[the remark this answers is not in the document]
Comment (aratidgr8) [reply includes/ECommerce.php]: Replace inline style rules with css class

File: includes/Promotions.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+var node = document.getElementById("promotion_product_data");
+node.innerHTML = `<div style="width: 100%;height:350px">
+    <p style=" text-align: center;padding-top:40px;line-height: 20px; font-size: 16px; font-weight: 600;">Boost Your Online Store Sales</p>
+    <p style=" text-align: center;line-height: 20px; font-size: 13px;">Maximize your sales by creating effective </br> promotions and campaigns like:</p>
+    <div style="text-align: center;">
+      <div style="display: flex;flex:1; align-items: center; justify-content:  space-around ;margin:0 auto; width: 50%;padding:20px 0">
+        <div style=" text-align: center;">
+          <svg width="49" height="48" viewBox="0 0 49 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (aratidgr8) [inline includes/Promotions.js:8]: Add translations for texts

File: includes/ECommerce.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (manikantakailasa) [reply includes/ECommerce.php]: done

File: includes/Promotions.js (hunk 3 above)
Comment (manikantakailasa) [reply includes/Promotions.js:8]: done

**Review by ramyakrishnai — APPROVED** [review]

**Comment by aratidgr8** [comment]
@manikantakailasa  Create a separate JIRA with all the text we will need to add translations. Also cypress tests are failing check once

**Review by aratidgr8 — DISMISSED** [review]

**Comment by ramyakrishnai** [comment]
As the changes have been addressed in https://github.com/newfold-labs/wp-module-ecommerce/pull/237, I am closing this PR.

**Pull request closed without merging**